src/Core/NationalCloud.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.
+ * Licensed under the MIT License.  See License in the project root
+ * for license information.
+ */
+
+namespace Microsoft\Graph\Core;
+
+/**
+ * Class NationalCloud
+ *
+ * Defines Graph hosts for the various national clouds
+ *
+ * @package Microsoft\Graph\Core
+ * @copyright 2021 Microsoft Corporation
+ * @license https://opensource.org/licenses/MIT MIT License
+ * @link https://developer.microsoft.com/graph
+ */
+final class NationalCloud
+{
+    const GLOBAL = "https://graph.microsoft.com";
```

Comment:
Please take into account only the hostnames. 
A URL with some port number like https://graph.microsoft.com:1234/v1.0 would be a valid format to pass in. 
Therefore, the National Cloud array should contain only the hosts  and verify if the url contains the valid host. 
Here is the implementation in JS for reference: 
https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/dev/src/GraphRequestUtil.ts

Reply:
Thanks for pointing this out. Adding this functionality

Reply:
Adding this.